config/default/jenkins-infra.yaml

```diff
@@ -297,235 +297,95 @@ jenkins:
-                      scanCredentialsId('github-app-infra')
-                      repoOwner('jenkins-infra')
-                      repository('charts')
-                    }
-                  }
-                  factory {
-                    workflowBranchProjectFactory {
-                      scriptPath('Jenkinsfile_k8s')
-                    }
-                  }
-                  orphanedItemStrategy {
-                    discardOldItems {
-                      daysToKeep(1) // Keep removed SCM heads/branch/PRs only for 1 day
-                    }
-                  }
-                  configure { node ->
-                    def traits = node / 'sources' / 'data' / 'jenkins.branch.BranchSource' / 'source' / 'traits'
-                    traits << 'org.jenkinsci.plugins.github__branch__source.BranchDiscoveryTrait' {
-                      strategyId(1)
-                    }
-                    traits << 'org.jenkinsci.plugins.github__branch__source.OriginPullRequestDiscoveryTrait' {
-                      strategyId(1)
-                    }
-                    traits << 'org.jenkinsci.plugins.github__branch__source.ForkPullRequestDiscoveryTrait' {
-                      strategyId(1)
-                      trust(class: 'org.jenkinsci.plugins.github_branch_source.ForkPullRequestDiscoveryTrait$TrustPermission')
-                    }
-                    traits << 'jenkins.scm.impl.trait.WildcardSCMHeadFilterTrait' {
-                      includes('main master PR-*')
-                      excludes()
-                    }
-                    traits << 'org.jenkinsci.plugins.github.label.filter.PullRequestLabelsBlackListFilterTrait' {
-                      labels('on-hold ci-skip')
-                    }
-                  }
-                }
-            - script: >
-                multibranchPipelineJob('plugin-site') {
-                  displayName "Plugin Site"
-                  branchSources {
-                    github {
-                      id('2019081602')
-                      scanCredentialsId('github-app-infra')
-                      repoOwner('jenkins-infra')
-                      repository('plugin-site')
-                      includes('master')
-                    }
-                  }
-                  factory {
-                    workflowBranchProjectFactory {
-                      scriptPath('Jenkinsfile_k8s')
-                    }
-                  }
-                }
-            - script: >
-                organizationFolder('Docker Builds') {
-                  description('Docker Builds')
-                  displayName('Docker Builds')
-                  organizations {
-                    github {
-                      repoOwner("jenkins-infra")
-                      apiUri("https://api.github.com")
-                      credentialsId('github-app-infra')
+                [
+                  ['Docker Builds', 'jenkins-infra', 'docker-.*|^jenkins.io$'],
```

Comment:
Is there any reason to use a list of arrays instead of a list of map? I find the latter one easier to read, but it's not blocker

Reply:
Good amelioration proposal, it would help the reader! 👍

Reply:
Done in #1126